rr_computer_vision/src/computer_vision_main.cpp

```diff
@@ -19,7 +18,9 @@ int main(int argc, char** argv)
   ros::init(argc, argv, "rr_computer_vision");
   ros::NodeHandle nh;
   ROS_INFO("Starting Computer Vision Node");
+  
   ComputerVision computer_vision(nh);
+  // EndlineDetection ec(nh);
```

Comment:
Why is this commented out if endline works?

Reply:
Cuz in this main function, everything should eventually be setup through a rosparam indicating the race type, and that will instantiate the necessary objects for that race. Also to avoid overhead of all the callbacks from this running when people branch out from develop